src/services/TechRecordsService.ts

```diff
@@ -215,48 +217,62 @@ class TechRecordsService {
       });
   }
 
-  private createAndArchiveTechRecord(techRecord: ITechRecordWrapper, msUserDetails: any) {
-    const {statusCode} = techRecord.techRecord[0];
-    delete techRecord.techRecord[0].statusCode;
+  public createAndArchiveTechRecord(updatedTechRecord: ITechRecordWrapper, msUserDetails: IMsUserDetails, oldStatusCode: STATUS | undefined) {
+    const {statusCode} = updatedTechRecord.techRecord[0];
+    delete updatedTechRecord.techRecord[0].statusCode;
+    const isPayloadValid = validatePayload(updatedTechRecord.techRecord[0]);
     if (statusCode === STATUS.ARCHIVED) {
```

Comment:
should be oldStatusCode

Reply:
It can be both oldStatusCode and statusCode.  If Old has the value that means the status of an archived record was updated otherwise its not a status change but some other attributes may be updated.

Reply:
what happens if oldStatusCode is archived and statusCode(the updated one) is current or provisional? That will basically unarchive a record and that's not allowed. Once a record is archived it cannot be updated in any way.